pyproject.toml

```
@@ -14,7 +14,7 @@ classifiers = [
 description = "Ophyd devices and other utils that could be used across DLS beamlines"
 dependencies = [
     "ophyd",
-    "ophyd_async@git+https://github.com/bluesky/ophyd-async@ec5729640041ee5b77b4614158793af3a34cf9d8", #Use a specific branch from ophyd async until https://github.com/bluesky/ophyd-async/pull/101 is merged
+    "ophyd_async@git+https://github.com/bluesky/ophyd-async",
```

Try this, think it might fix the container build, `ophyd-async` is the correct package name. Not sure why `ophyd_async` works at all to be honest.

```suggestion
    "ophyd-async@git+https://github.com/bluesky/ophyd-async",
```

---

Looks like it was ophyd-async prior to being pinned, and the underscore change was hidden by the direct git dependency

---

Does anyone know if it _needs_ to be pinned to main anyway? @DiamondJoseph @DominicOram

---

container build doesn't work with main, needs to be pointing directly at a SHA. This is fixed in the copier template